README.md

```diff
@@ -57,3 +58,9 @@ Feel free to send a PR! :smile:
 - Fix a [bug](https://github.com/npm/npm/issues), get a fashionable pair of socks! [Reference](http://blog.npmjs.org/post/129827785565/npm-weekly-30-package-scripts-for-tooling-a)
 
 ![](https://partners.npmjs.com/weekly/weekly30/socks-1100x.jpg)
+
+## Raygun
+
+- Simply create an account, integrate Raygun Pulse or Crash Reporting with your website or app, send data and we'll post you out a free Raygun Kitset. Easy. [Reference](https://raygun.com/swag/kitset)
+
+<img src="https://scontent-lht6-1.cdninstagram.com/vp/088a01f8c570b091734ac594489b6cf1/5B19AEDC/t51.2885-15/e35/10544295_219842845030468_1850581895_n.jpg" width="250"></img>
```

Comment:
Github markdown supports image sizing 
`![](https://scontent-lht6-1.cdninstagram.com/vp/088a01f8c570b091734ac594489b6cf1/5B19AEDC/t51.2885-15/e35/10544295_219842845030468_1850581895_n.jpg | width=50)`

Reply:
Ah, brilliant. Thank you. I did have a quick google but it seems I wasn't looking very hard.

Reply:
Unfortunately, this doesn't seem to work. I did a quick Google search and people seem to agree, the above method no longer works for some reason. I also tried using the `&s` parameter to no avail. 
I've reverted back to using HTML for now. If you know of any other ways this can be done in MD i'll be happy to try.

Reply:
oh thats too bad :disappointed: